internal/daemon/controller/handlers/workers/worker_service.go

```diff
@@ -612,6 +668,45 @@ func (s Service) authResult(ctx context.Context, id string, a action.Type) auth.
 	return auth.Verify(ctx, opts...)
 }
 
+func (s Service) listCertificateAuthorityFromRepo(ctx context.Context) (*types.RootCertificates, error) {
+	repo, err := s.workerAuthFn()
+	if err != nil {
+		return nil, err
+	}
+
+	certs := &types.RootCertificates{Id: server.CaId}
+	err = repo.Load(ctx, certs)
+	if err != nil {
+		return nil, err
+	}
+
+	return certs, nil
+}
+
+func certificateAuthorityToProto(in *types.RootCertificates) *pb.CertificateAuthority {
+	certs := make([]*pb.Certificate, 2)
+
+	next := in.GetNext()
```

Comment:
Any reason you return Next first? I would think we'd want to go in chronological order, which would mean returning Current first.

Reply:
No reason! I'll swap it so current is returned first.